pyvrp/tests/test_ProblemData.py

```diff
@@ -117,34 +117,47 @@ def test_raises_for_invalid_depot_data(
 
     with assert_raises(ValueError):
         ProblemData(
-            clients=[depot],
+            clients=[],
+            depots=[depot],
             vehicle_types=[VehicleType(1, 2)],
             distance_matrix=np.asarray([[0]], dtype=int),
             duration_matrix=np.asarray([[0]], dtype=int),
         )
 
 
-def test_problem_data_raises_when_no_clients_provided():
+def test_problem_data_raises_when_not_exactly_one_depot_is_provided():
     """
     Tests that the ``ProblemData`` constructor raises a ``ValueError`` when
-    no clients are provided.
+    not exactly one depot is provided.
     """
     with assert_raises(ValueError):
         ProblemData(
             clients=[],
+            depots=[],
```

Comment:
So just to check not having a depot is not allowed but not having clients is ok?

Reply:
Yes